db/schemas/schemas.py

```diff
@@ -15,3 +17,16 @@ class Meta:
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+class JustScoreMetadata(SQLAlchemyAutoSchema):
+    """JustScoreMetadata The marshmallow class used to turn SQLAlchemy
+    rows into json for return in http responses."""
+
+    class Meta:
+        model = JustScore
+        include_relationships = True
```

Comment:
❓ What relationship are we including here?

Reply:
I think by default it includes all relationships within the model which for this case is the foreign key relationship with assessment records 👍🏾